src/test/java/org/jabref/logic/autosaveandbackup/BackupManagerTest.java

```diff
@@ -15,4 +17,22 @@ public void backupFileNameIsCorrectlyGeneratedWithinTmpDirectory() {
         Path savPath = BackupManager.getBackupPath(bibPath);
         assertEquals(Paths.get("tmp", "test.bib.sav"), savPath);
     }
+
+    @Test
+    public void backupFileIsEqualForNonExistingBackup() throws Exception {
+        Path originalFile = Path.of(BackupManagerTest.class.getResource("no-backup.bib").toURI());
+        assertFalse(BackupManager.backupFileDiffers(originalFile));
+    }
+
+    @Test
+    public void backupFileIsEqual() throws Exception {
+        Path originalFile = Path.of(BackupManagerTest.class.getResource("no-changes.bib").toURI());
+        assertFalse(BackupManager.backupFileDiffers(originalFile));
+    }
+
+    @Test
+    public void backupFileDiffers() throws Exception {
+        Path originalFile = Path.of(BackupManagerTest.class.getResource("changes.bib").toURI());
```

Comment:
How do these tests work? I cannot see a `.sav` file in the folder?!

Reply:
Save files are missing in the commit! @tobiasdiez Why do you approve the PR if you have objections (that are correct!)?

Reply:
They did not work - I added the missing files in ee472cf

![grafik](https://user-images.githubusercontent.com/1366654/76710270-64649700-6706-11ea-85a6-2f73a34559ee.png)